app/models/post.rb

```diff
@@ -27,7 +27,7 @@ def to_s
     title
   end
 
-  def printLimit
+  def print_limit
```

Comment:
Perhaps add a comment to explain why it is so complicated. Or consider moving it to a helper if it is only used in views.